scio-google-cloud-platform/src/main/java/com/spotify/scio/bigtable/BigtableBulkWriter.java

```diff
@@ -142,17 +142,13 @@ public void processElement(@Element Iterable<KV<ByteString, Iterable<Mutation>>>
 
     @FinishBundle
     public void finishBundle() throws Exception {
-      bigtableWriter.flush();
```

Comment:
is the `flush` call no longer needed?

Reply:
flush is not part of the API anymore. See this commit: https://github.com/apache/beam/commit/b566480cec6d413efa0d62711b3f3f9ca7a0f6fd